ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace zadanie10
+{
+    internal class Program
+    {
+        static string Print(int[] n)
+        {
+            string str = "";
+            for (int i = 0; i < n.Length; i++)
+            {
+                str += $"{n[i]} ";
+            }
+            return Regex.Replace(str, ".{" + 20 + "}(?!$)", "$0\n");
+        }
+
+        static int[] EachFourth(int[] n)
+        {
+            int count = 4;
+            for (int i = 0; i < n.Length; i++)
+            {
+                if (i == count)
+                {
+                    n[i] = 9 - i;
+                    count += 4;
+                }
+            }
+            return n;
+        }
+        static int CountM(int[] n, int m)
+        {
+            int count = 0;
+            foreach (int i in n)
+            {
+                if (i >= 0 && i <= 9)
+                {
+                    if (i == m)
+                    {
+                        count++;
+                    }
+                }
+            }
+            return count;
+        }
```

Comment:
Здесь слишком много необязательных фигурных скобок
```suggestion
        static int CountM(int[] n, int m)
        {
            int count = 0;
            foreach (int i in n)
                if (i >= 0 && i <= 9)
                    if (i == m)
                        count++;
                        
            return count;
        }
```